src/js/components/TextInput/README.md

```diff
@@ -183,10 +183,12 @@ node
 **plain**
 
 Whether this is a plain input with no border or padding.
-      Only use this when the containing context provides sufficient affordance
+      Only use this when the containing context provides sufficient affordance.
```

Comment:
I think the first line isn't accurate if it's referring to the boolean value
> Whether this is a plain input with no border or padding.
Should it be something like:
```
Whether this is a plain input with no border or outline.
Only use this when the containing context provides sufficient affordance.
Use "full" to remove the padding in addition to removing the border and outline.
```
Thoughts?

Reply:
@jcfilben where are we with this comment?

Reply:
I updated the readme to say 'no border or outline.' instead of 'no border or padding'